tasks/inventory_report/schemas/input.json

```diff
@@ -32,14 +33,14 @@
       "startTimestamp": {
         "description": "Start time for cumulusCreateTime date range to compare data.",

```

Comment:
Wondering since this is input from Cumulus if we should adjust the description to call out the Cumulus `createdAt` field instead of our database field of `cumulus_create_time`, or maybe mention both for these two time fields?

Reply:
Probably best to leave our internal DB out of the schema. Updated.